src/Modules/StringExtensions.bas

```diff
@@ -23,8 +25,10 @@ Public Function TrimRight(text As String, c As String) As String
     Dim textLength As Long
     textLength = Len(text)
     
+    Dim lastCharacter As String
+        
     Do While textLength > 0
-        Dim lastCharacter As String
+        

```

Comment:
same here, please remove the additional whitespace

Reply:
Maybe I'm not paying attention... about the additional spaces in what line is this about?

Reply:
I'm talking about [this](https://github.com/michaelneu/webxcel/blob/c65604349e4281b5fb5cdf28373e0a1b4a07bec5/src/Modules/StringExtensions.bas#L9) and [this](https://github.com/michaelneu/webxcel/blob/c65604349e4281b5fb5cdf28373e0a1b4a07bec5/src/Modules/StringExtensions.bas#L31) newline, which look kinda awkward.